autorest/azure/auth/auth.go

```diff
@@ -38,7 +38,7 @@ import (
 // 1. Client credentials
 // 2. Client certificate
 // 3. Username password
-// 4. MSI
+// 4. IMDS
```

Comment:
>IMDS [](start = 6, length = 4)

is still through MSI, IDMS is just a service that exists on every VM in Azure

Reply:
Fixed